test/boost/flaky_test.cc

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (C) 2023-present ScyllaDB
+ */
+
+/*
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+#include <seastar/testing/test_case.hh>
+#include "test/lib/random_utils.hh"
+#include <fmt/core.h>
+
+SEASTAR_TEST_CASE(flaky_test) {
```

Comment:
or, we can add a `expected_failures(1)` decorator to the flaky test? like
```c++
SEASTAR_TEST_CASE(flaky_test, 
    boost::unit_test::expected_failure(1)) {
  // ...
}
```
see https://www.boost.org/doc/libs/master/libs/test/doc/html/boost_test/testing_tools/expected_failures.html

i understand this is not the same as run-until-success though..